cime/config/e3sm/machines/config_machines.xml

```diff
@@ -3017,9 +2740,9 @@
       <env name="HDF5_PATH">$ENV{OLCF_HDF5_ROOT}</env>
       <env name="PNETCDF_PATH">$ENV{OLCF_PARALLEL_NETCDF_ROOT}</env>
     </environment_variables>
-    <environment_variables mpilib="!mpi-serial" compiler="pgiacc">
-      <env name="OMPI_CXX">$SRCROOT/externals/kokkos/bin/nvcc_wrapper</env>
-    </environment_variables>
+    <!-- <environment_variables mpilib="!mpi-serial" compiler="pgigpu"> -->
```

Comment:
Why is this commented out and not just removed?

Reply:
It seems this is needed for kokkos. @amametjanov commented about this above:
"OMPI_CXX was pointing mpiCC wrapper to nvcc instead of pgc++ and causing SCORPIO build fail. We can add mpiCC-to-nvcc binding for Kokkos builds in CMake at a later time"
So it seems to be something we need, but it needs to be fixed down the road, so it made sense to leave it in.